clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -102,7 +114,15 @@ interface NexusIqService {
         val reference: String,
         val severity: Float,
         val url: URI?
-    )
+    ) {
+        /**
+         * Return an identifier for the scoring system used for this issue. According to the documentation, the
+         * prefix of the reference determines the scoring system.
+         */
```

Comment:
Here, it might also be convenient to have a reference/URL to the documentation.

Reply:
This was my intension originally, but the link is so long that it exceeded the allowed line length. I now use an abbreviated form.

Reply:
FYI, long http links are excempted from detekt's line length limit check.